In-house patch to remove unnecessary dependencies from Ironic's
requirements files. The specific reasons are as follows:

lxml    Not applicable

websockify  Not applicable

oslo.rootwrap Not applicable to Solaris

--- ironic-2015.1.2/requirements.txt.orig	2015-10-19 13:49:06.006805428 -0400
+++ ironic-2015.1.2/requirements.txt	2015-10-19 13:49:43.836513611 -0400
@@ -5,7 +5,6 @@
 SQLAlchemy<=0.9.99,>=0.9.7
 alembic<0.8.1,>=0.7.2
 eventlet!=0.17.0,>=0.16.1
-lxml>=2.3
 WebOb>=1.2.3
 greenlet>=0.3.2
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
@@ -18,12 +17,10 @@
 python-swiftclient<2.5.0,>=2.2.0
 stevedore<1.4.0,>=1.3.0 # Apache-2.0
 pysendfile==2.0.0
-websockify<0.7,>=0.6.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.context<0.3.0,>=0.2.0 # Apache-2.0
 oslo.db<1.8.0,>=1.7.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 oslo.policy<0.4.0,>=0.3.1 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
--- ironic-2015.1.2/ironic.egg-info/requires.txt.orig	2015-10-19 13:50:49.380390704 -0400
+++ ironic-2015.1.2/ironic.egg-info/requires.txt	2015-10-19 13:51:20.655399790 -0400
@@ -2,7 +2,6 @@
 SQLAlchemy<=0.9.99,>=0.9.7
 alembic<0.8.1,>=0.7.2
 eventlet!=0.17.0,>=0.16.1
-lxml>=2.3
 WebOb>=1.2.3
 greenlet>=0.3.2
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
@@ -15,12 +14,10 @@
 python-swiftclient<2.5.0,>=2.2.0
 stevedore<1.4.0,>=1.3.0 # Apache-2.0
 pysendfile==2.0.0
-websockify<0.7,>=0.6.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.context<0.3.0,>=0.2.0 # Apache-2.0
 oslo.db<1.8.0,>=1.7.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 oslo.policy<0.4.0,>=0.3.1 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
